package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -274,46 +275,63 @@ def __init__(self, filename, convert_units=True, **kwargs):
         self.units['velocity'] = kwargs.pop('velocityunit',
                                  self.units['length']+'/'+self.units['time'])
 
-    def _write_atoms(self, atoms, data):
+    def _write_atoms(self, atoms, data, atom_style):
```

Comment:
Forgive my LAMMPs ignorance, but what style are we currently writing? We should use that as the default.